app/forms/security_event_form.rb

```diff
@@ -279,6 +279,7 @@ def identity_from_agency_identity
     )
   end
 
+  # Not changing; this is inbound RISC calls
```

Comment:
let's remove this comment? since it's useful in the scope of this PR, but as a permanent comment, not really
```suggestion
```

Reply:
> it's useful in the scope of this PR, but as a permanent comment, not really

I was planning on leaving these comments in while I work, and then pulling them all out when I take this out of draft status. Right now it's a note-to-self as I go through these. Agree 💯 that it should come out before this lands, though.